rest/src/main/kotlin/builder/message/modify/MessageModifyStateHolder.kt

```diff
@@ -30,4 +29,5 @@ internal class MessageModifyStateHolder {
 
     var components: Optional<MutableList<MessageComponentBuilder>> = Optional.Missing()
 
+    var suppressEmbeds: OptionalBoolean = OptionalBoolean.Missing
```

Comment:
this isn't needed, we don't use the optional value directly but instead work with the nullable boolean to calculate the flags to send, `override var suppressEmbeds: Boolean? = null` is easier to read than `override var suppressEmbeds: Boolean? by state::suppressEmbeds.delegate()`

the same goes for `flags` which could also be removed from `MessageModifyStateHolder`